### PR TITLE
Add djlint

### LINKED ISSUE #25: Add djlint
### Motivation
In order to have HTML clean and standardized templates automatically, we should think about adding a linter and formatter for templates

### User Story
I, as a programmer, want to have consistent and clean HTML code from a reliable source

### Proposed Solution

Add djlint as formatter
- decide on configuration
- add a tool for it

### Alternatives
Do it manually (not recommended)

### Additional Context

None

### Screenshots

None

### PR DESCRIPTION
**Pull request by JoeyStk**
### Short description
Add djlint

### Proposed changes

- Add djlint to pyproject.toml
- Add djlint to pre-commit
- Add djlint as tool
- Add djlint Circle CI job

### Side effects

- Rebasing after PR #26  has been merged

### Resolved issues

Fixes: #25

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,6 +55,20 @@ jobs:
...
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: Enable virtual environment
+          command: echo "source .venv/bin/activate" >> $BASH_ENV
+      - run:
+          name: Check formatting of Django templates
+          command: djlint --check --lint integreat_cms
```
Comment (timobrembeck) [inline .circleci/config.yml]:
```suggestion
          command: djlint --check --lint integreat_compass
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -14,3 +14,7 @@ repos:
     rev: v2.7.1
     hooks:
       - id: prettier
+  - repo: https://github.com/Riverside-Healthcare/djLint
+    rev: v1.19.6
```
Comment (timobrembeck) [inline .pre-commit-config.yaml]:
```suggestion
    rev: v1.23.1
```

File: pyproject.toml [hunk 3]
```diff
@@ -105,6 +119,9 @@ ignore-paths = [
 fail-on = "I"
 django-settings-module="integreat_compass.core.settings"
 
+[tool.djlint]
+profile="django"
+
```
Comment (timobrembeck) [inline pyproject.toml]: I would move this up, so it's not in between the pylint settings.

**Review by timobrembeck — CHANGES_REQUESTED** [review]

**Review by timobrembeck — APPROVED** [review]

File: .circleci/config.yml [hunk 4]
```diff
@@ -69,6 +69,20 @@ jobs:
       - run:
           name: Check black code style
           command: black --check .
+  djlint:
```
Comment (timobrembeck) [inline .circleci/config.yml:75]: Ah wait, this job is not contained in the dev workflow, needs to be added there as well.

**Review by timobrembeck — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 4 above)
Comment (JoeyStk) [reply .circleci/config.yml:75]: This should be fixed now :)

**Review by timobrembeck — CHANGES_REQUESTED** [review]
Thanks, could you resolve the git conflicts?

**Comment by JoeyStk** [comment]
> Thanks, could you resolve the git conflicts?

Yes. While rebasing, I sorted the packages alphabetically, I hope I didn't mess that up

**Review by timobrembeck — APPROVED** [review]

**Pull request merged**